src/docs/data_usage_telemetry.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Data Usage and Telemetry
+---
+
+# Data Usage and Telemetry in Theia
+
+Eclipse Theia and Theia IDE are fully open-source projects. This means you can review the source code to verify exactly how the application behaves. Transparency is a core principle - no hidden services or undisclosed data collection take place.
+
+## No Built-in Telemetry
+
+The Theia IDE itself does not include built-in telemetry. It does not automatically collect or transmit user activity, project details, or code.
+
+However, tool builders who create custom products based on Theia can choose to add their own telemetry systems. If you are using a customized Theia-based product, you should consult that product's specific privacy policy and documentation for details about its data collection practices.
+
+## Automatic Network Connections
+
+While Theia IDE does not include telemetry, there are two automatic checks made during normal operation:
+
+- **Update Check for Theia IDE**: The IDE may query the download servers to see if a newer version of Theia IDE is available.
+- **Extension Update Check**: The IDE may query Open VSX to determine whether newer versions of installed extensions are available.
+
+These checks are limited to update metadata and do not transmit user code or workspace data.
+
+## AI Features and Data Handling
+
+Theia AI and related AI features are opt-in. If you do not enable them, no AI-related data is sent anywhere.
+
+When Theia AI features are enabled:
+
+### Manual Activation
+- Most AI interactions (e.g., chat, code transformation) are triggered explicitly by the user.
+- Automatic code completion is separately opt-in to prevent accidental transmission of code.
```

Comment:
Note that we still assign a key shortcut which is active by default. So a user could still accidentally transmit data when AI is enabled although they didn't want to

Reply:
At the moment we do not offer to turn off AI autocompletion (except for disabling *all* AI features). The only two options are: automatic code completion or code completion via keybinding.

Reply:
Why not restructure to have a setting for "AI code completion" and a sub-setting "Complet automatically"? Turning off code completion turns it off for both manual and automatic. Kind of like we can turn off all AI features at once.

Reply:
Yes that would make sense. In the past we used an enum, so that would also be an option: off / manual / automatic